plugins/outputs/cloudrun/README.md

```diff
@@ -0,0 +1,19 @@
+### CloudRun
```

Comment:
This README is a bit short. Please at least state what you can do with the plugin. Which service is it intented for (maybe add a link)? What are current limitations? Cove some further explanations for the config parameters... Take a look at other REAMEs in the `plugins/outputs/` folder e.g. the [cloudwatch one](https://github.com/influxdata/telegraf/tree/master/plugins/outputs/cloudwatch)...